WebJobs.Script.proj

```diff
@@ -168,6 +168,11 @@
     <xunit Assemblies="@(UnitTestAssemblies)" ExcludeTraits="$(ExcludedTestTraits)" />
   </Target>
 
+  <Target Name="MochaTest" AfterTargets="TestBuild">
+    <Exec Command="npm i mocha chai mocha-appveyor-reporter" />
```

Comment:
Looks like that's already the case but I'm not sure how (if) AppVeyor sorts the tests.